extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ResourceNotFoundData.java

```diff
@@ -124,8 +124,12 @@ public String getHTMLContent() {
 
         // Additional Endpoints
         if (!this.additionalEndpoints.isEmpty()) {
+            List<AdditionalRouteDescription> endpoints = this.additionalEndpoints.stream().sorted(
```

Comment:
since it's the same for all 3 media types, you can probably extract this logic into a private method